modin/data_management/factories/dispatcher.py

```diff
@@ -11,20 +11,47 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""
+Contain IO dispatcher class.
+
+Dispatcher routes the work to excecution engine-specific functions.
```

Comment:
```suggestion
Dispatcher routes the work to execution backend-specific functions.
```

Reply:
I think that your change is right, but further in the docs of this file, and what's more important - further in the code, we're always using the term "engine". I know that it's actually a backend, that's why I use "backend" term in 'rst' docs. 

To avoid mixing the terms "engine" and "backend" in the actual code and docstrings, I would prefer to keep it as is, and then change it in the PR, which will also change the name on entities in the code of factories module.

Reply:
Okay, fix that in another PR, please.